kart/helper.py

```diff
@@ -203,9 +204,11 @@ class struct_semun(ctypes.Union):
                         )
 
                     except Exception as e:
+                        print(os.environ)
```

Comment:
Secrets could be in the environment, we shouldn't be printing values at all — maybe just keys?

Ideally we'd gate this env info, and the full traceback, behind a verbosity flag.

(And `print()` doesn't go to stderr by default)

Reply:
Yep, that shouldn't be there, debug.